src/test/ui/const-generics/unused-substs/unused-substs-4.rs

```diff
@@ -0,0 +1,11 @@
+#![feature(const_generics)] //~ WARN the feature `const_generics` is incomplete
```

Comment:
I'd like to see a FIXME comment indicating that this is a case we would like to work but which doesn't presently.